### PR TITLE
Updated contributing.md for sistent documentation

### PR DESCRIPTION
**Pull request by Vidit-Kushwaha**
**Description**

This PR updates the `CONTRIBUTING.md` file to include instructions on contributing to the [Sistent Library Components](https://layer5.io/projects/sistent/components). It outlines the process of selecting the area/sistent label in the [GitHub issue tracker](https://github.com/layer5io/layer5/labels/project%2Fsistent) and explains the new dynamic page creation workflow.

**Notes for Reviewers**
- Added a section to CONTRIBUTING.md for Sistent Library contributions.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ac42648e501bfc6bdac9525be92c1da30985e0f4 at: https://6756de960526090141192f9f--layer5.netlify.app

**Review by leecalcote — APPROVED** [review]
LGTM. 

@nebula-aac, do you agree?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -129,6 +130,21 @@ If you'd like to contribute a post to layer5.io/blog, please open an Issue and s
...
+### Adding Sistent Component
+1. All content is managed in a centralized file:
+   ```
+   src/sections/Projects/Sistent/components/content.js
+   ```
+2. To add a new page, simply update this `content.js` file with the necessary details.
+3. The system will dynamically generate pages based on this content and handle routing automatically.
+
+### Example
+Refer to the **Button component** in the Sistent Library for an example of how to structure the content.
```
Comment (sudhanshutech) [inline CONTRIBUTING.md:146]: add a link to this button component

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -129,6 +130,21 @@ If you'd like to contribute a post to layer5.io/blog, please open an Issue and s
 3. Follow the instructions included in the news template and name the new file after the title of the news article.
 4. Entries will be listed in chronological order automatically.
 
+## Contributing to Layer5's Sistent documentation
+
+If you'd like to contribute a post to layer5.io/projects/sistent/components. start by selecting the area/sistent label in the [#GitHub issue tracker](https://github.com/layer5io/layer5/labels/project%2Fsistent). We've streamlined the process by introducing a dynamic page creation workflow, simplifying the addition of new pages and ensuring a consistent structure for all contributions.
```
Comment (sudhanshutech) [inline CONTRIBUTING.md:135]: First: don't write the link on the documentation(use link tag and use label here)
second: why its specific to components? anywhere near sistent project should be right? and this particular process could be for adding the components for sistent.

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Vidit-Kushwaha) [reply CONTRIBUTING.md]: I have updated contributing.md with label and general guidelines for sistent

**Comment by l5io** [comment]
🚀 Preview for commit 6d7df3a07495c89354d257ac394c7686c0349640 at: https://675ac59060b1832ca4c8662c--layer5.netlify.app

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**